### ISSUE TITLE
a11y:  "required" tag next to Request Body is not accessible

### ISSUE THREAD

**Issue by ghost** [swagger-api/swagger-ui#9572]
### Q&A (please complete the following information)
 - OS: [e.g. macOS]  Windows 10
 - Browser: [e.g. chrome, safari]  Chrome
 - Version: [e.g. 22]  121
 - Method of installation: [e.g. npm, dist assets]  npm
 - Swagger-UI version: [e.g. 3.10.0]   Swagger UI React 5.11.3
 - Swagger/OpenAPI version: [e.g. Swagger 2.0, OpenAPI 3.0]  OpenAPI 3.0

### Content & configuration
Basic usage, no special configuration

### Describe the bug you're encountering
We use Sort Site tool to evaluate our site for accessibility.  It is flagging that the "required" tag next to the Request Body is not accessible:  "Content inserted with CSS is not available to people who turn off style sheets or use custom styles.  Move the content from CSS to the HTML page so that all users can see it".  Violates WCAG 2.0 A F87 https://www.w3.org/TR/WCAG20-TECHS/F87.html
This is happening in an h4 with classes "opblock-title parameter__name required"

In addition, this "required" tag is too small a font and the red is not dark enough to provide enough color contrast, both of which make it hard to read for those with visibility issues.

### To reproduce...
Click to expand a method pane, e.g. a POST method, with a request body.  Next to "Request body" is a very small "required" label.

### Expected behavior
The "required" tag should be easily visible

### Screenshots
![image](https://github.com/swagger-api/swagger-ui/assets/86489380/f0201aba-4814-4021-b224-0acf910db954)

### Additional context or thoughts
Here's the Sort Site output with the error:
![image](https://github.com/swagger-api/swagger-ui/assets/86489380/271500ef-234f-4a0f-9b4f-e192a6e91393)

**Comment by mgifford** [comment]
It would be nice if this were more flexible. You want it to be consistent with however the API is being shown.  Users don't want to learn the DKAN & Swagger UI's.  Certainly, there is a reasonably well tested model within Drupal. Definitely the "required" text is too small.
